libraries/Microsoft.Bot.Connector/Authentication/AuthenticationConstants.cs

```diff
@@ -10,6 +10,11 @@ namespace Microsoft.Bot.Connector.Authentication
     /// </summary>
     public static class AuthenticationConstants
     {
```

Comment:
I would probably make this a sealed class, so nobody can derive from it and change constants

Reply:
Static classes cannot be sealed...you get one or the other :/